docs/getting-started-with-scalardb.md

```diff
@@ -272,6 +300,51 @@ $ ../../gradlew run --args="-mode transaction -action charge -amount 0 -to merch
 $ ../../gradlew run --args="-mode transaction -action pay -amount 100 -to merchant1 -from user1"
 ```
 
+## Using JDBC transaction
+
+When you use a JDBC database, you can choose the transaction manager type you use.
```

Comment:
```suggestion
When you use a JDBC database as a backend database, you can optionally use the native transaction manager of a JDBC database instead of the default `ConsensusCommit` transaction manager.
```